package.json

```diff
@@ -23,10 +23,12 @@
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-eslint-plugin": "^2.2.1",
     "eslint-plugin-mocha": "^7.0.0",
+    "eslint-plugin-prettier": "^3.1.3",
     "fs-extra": "^9.0.0",
     "husky": "^4.2.5",
     "lerna": "^3.21.0",
     "open-cli": "^6.0.1",
+    "prettier": "^2.0.5",
```

Comment:
Is there a possibility to reuse `prettier` from `@loopback/build`?

Reply:
At minimum, please update https://github.com/strongloop/loopback-next/blob/master/bin/sync-dev-deps.js#L42 to include `prettier` so that we can ensure the same versions of `prettier` being used.

Reply:
@dougal83 Since you are here, maybe we should use `.prettierrc.js` so that it can reuse https://github.com/strongloop/loopback-next/blob/master/packages/build/config/.prettierrc.

See https://prettier.io/docs/en/configuration.html#basic-configuration

Reply:
+1 to update `sync-dev-deps` to ensure we have the same version of `prettier` in monorepo root and in the build package.

We already have a `.prettierrc` file in our monorepo root, this pull request is not changing that in any way. IMO, reworking the monorepo-level prettier config to inherit from `@loopback/build` is out of scope of this pull request.

Reply:
I'm fine to leave `.prettyrc` refactoring out.